stages/org.osbuild.systemd.unit.create.meta.json

```diff
@@ -27,6 +27,10 @@
     "    - 'ExecStart' - [string]",
     "    - 'Environment' - [object]",
     "    - 'EnvironmentFile' - [string]",
+    "  - 'Mount' section",
+    "    - 'What' - string",
+    "    - 'Where' - string",
+    "    - 'Type' - string",
```

Comment:
It feels like "Options" would be useful too

Reply:
Ah, i see you have it below, only not in the docs

Reply:
Starting to think this long description isn't worth keeping up-to-date :)